staging/src/k8s.io/client-go/rest/config.go

```diff
@@ -178,10 +187,21 @@ func (c *Config) String() string {
 	if cc.AuthConfigPersister != nil {
 		cc.AuthConfigPersister = sanitizedAuthConfigPersister{cc.AuthConfigPersister}
 	}
-
+	if cc.Exec.Config != nil {
+		cc.Exec.Config = sanitizedObject{Object: cc.Exec.Config}
+	}
 	return fmt.Sprintf("%#v", cc)
 }
 
+// Exec plugin authentication provider.
+type Exec struct {
+	// ExecProvider provides the config needed to execute the exec plugin.
+	ExecProvider *clientcmdapi.ExecConfig
+
+	// Config holds additional config data that is specific to the exec plugin with regards to the cluster being authenticated to.
+	Config runtime.Object
```

Comment:
Why isn't this a runtime.RawExtension?

Reply:
Because conversion automatically converts `runtime.RawExtension` into a `runtime.Object` (`*runtime.Unknown` IIRC).  I think it has to process it no matter what because it goes from YAML to JSON.

Reply:
Note that this is pre-existing:

https://github.com/kubernetes/kubernetes/blob/f3f666d5f1f6f74a8c948a5c64af993696178244/staging/src/k8s.io/client-go/tools/clientcmd/api/v1/types.go#L91 and https://github.com/kubernetes/kubernetes/blob/f3f666d5f1f6f74a8c948a5c64af993696178244/staging/src/k8s.io/client-go/tools/clientcmd/api/v1/types.go#L183 get converted to https://github.com/kubernetes/kubernetes/blob/f3f666d5f1f6f74a8c948a5c64af993696178244/staging/src/k8s.io/client-go/tools/clientcmd/api/types.go#L98